osquery/tables/system/darwin/asl.cpp

```diff
@@ -0,0 +1,32 @@
+#include <asl.h>
+
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "asl_utils.h"
```

Comment:
If you don't mind, we're a little weird here, but we assume the "local" is from the project root. I know it's not 'correct' but it's at least consistent:
```
#include "osquery/tables/system/darwin/asl_utils.h"
```